Formula/node.rb

```diff
@@ -56,30 +56,11 @@ def install
 
     if build.with? "npm"
       resource("npm").stage buildpath/"npm_install"
-
-      # make sure npm can find node
-      ENV.prepend_path "PATH", bin
-      # set log level temporarily for npm's `make install`
-      ENV["NPM_CONFIG_LOGLEVEL"] = "verbose"
-      # unset prefix temporarily for npm's `make install`
-      ENV.delete "NPM_CONFIG_PREFIX"
-
-      cd buildpath/"npm_install" do
-        system "./configure", "--prefix=#{libexec}/npm"
```

Comment:
To be honest, the way we using `./configure` especially was a very slight hack to begin with IMO. I mean, I appreciate the entire way we handle `npm` is a hack but 😄.

The problem here is that if you retain this process all you end up with in `libexec` is symlinks back to the `npm` in `buildpath`, which is presuming you even passed `--keep-tmp`. Without that they just be dead symlinks.

Even if you build in place, i.e. moving this to `libexec` and then running `make install` all it does is create symlinks from that directory to `bin` and `lib/node_modules`, essentially mandating the presence of `npm` always within the keg, which semi-works right up until the point you ever want to `unlink` and then `link` the `node` formula at some random future point.

If we went down that path we'd possibly have to physically find a way to block users from upgrading their own `npm` installations, and I'll let you guess how popular that would make us 🙈.